docs/src/main/sphinx/functions/binary.rst

```diff
@@ -172,7 +172,10 @@ Hashing functions
 
 .. function:: murmur3(binary) -> varbinary
 
-    Computes the 128-bit murmur3 hash of ``binary``.
+    Computes the 128-bit `MurmurHash3 <https://en.wikipedia.org/wiki/MurmurHash>`_ value of ``binary``::
```

Comment:
picking nits: can this line be wrapped at about 80 chars?

Reply:
Done. I will try to do better next time.

Reply:
> picking nits: can this line be wrapped at about 80 chars?
Done. I will try to do better next time.